test/e2e/antreapolicy_test.go

```diff
@@ -4350,32 +4216,36 @@ func TestAntreaPolicy(t *testing.T) {
 
 	initialize(t, data)
 
-	t.Run("TestGroupValidateAntreaNativePolicies", func(t *testing.T) {
-		t.Run("Case=ACNPNoPriority", func(t *testing.T) { testInvalidACNPNoPriority(t) })
-		t.Run("Case=ANPNoPriority", func(t *testing.T) { testInvalidANPNoPriority(t) })
-		t.Run("Case=ANPRuleNameNotUniqueDenied", func(t *testing.T) { testInvalidANPRuleNameNotUnique(t) })
-		t.Run("Case=ANPTierDoesNotExistDenied", func(t *testing.T) { testInvalidANPTierDoesNotExist(t) })
-		t.Run("Case=ANPPortRangePortUnsetDenied", func(t *testing.T) { testInvalidANPPortRangePortUnset(t) })
-		t.Run("Case=ANPPortRangePortEndPortSmallDenied", func(t *testing.T) { testInvalidANPPortRangeEndPortSmall(t) })
-		t.Run("Case=ANPIngressPeerGroupSetWithIPBlock", func(t *testing.T) { testInvalidANPIngressPeerGroupSetWithIPBlock(t) })
-		t.Run("Case=ANPIngressPeerGroupSetWithPodSelector", func(t *testing.T) { testInvalidANPIngressPeerGroupSetWithPodSelector(t) })
-		t.Run("Case=ACNPInvalidPodSelectorNsSelectorMatchExpressions", func(t *testing.T) { testInvalidACNPPodSelectorNsSelectorMatchExpressions(t, data) })
-	})
-
-	t.Run("TestGroupValidateTiers", func(t *testing.T) {
-		t.Run("Case=TierOverlapPriorityDenied", func(t *testing.T) { testInvalidTierPriorityOverlap(t) })
-		t.Run("Case=TierOverlapReservedTierPriorityDenied", func(t *testing.T) { testInvalidTierReservedPriority(t) })
-		t.Run("Case=TierPriorityUpdateDenied", func(t *testing.T) { testInvalidTierPriorityUpdate(t) })
-		t.Run("Case=TierACNPReferencedDeleteDenied", func(t *testing.T) { testInvalidTierACNPRefDelete(t) })
-		t.Run("Case=TierANPRefDeleteDenied", func(t *testing.T) { testInvalidTierANPRefDelete(t) })
-		t.Run("Case=TierReservedDeleteDenied", func(t *testing.T) { testInvalidTierReservedDelete(t) })
+	// This test group only provides one case for each CR, including ACNP, ANP, Tier,
```

Comment:
I personally feel like the tests can be further reduced here. If I'm not mistaken, any update validation would have already covered create validation?

Reply:
I put update event cases here to ensure update events could also call the webhook. Let me know if you think it's unnecessary.
And thanks for pointing this out. I just noticed that I didn't add update events for the mutation webhook. I will update PR based on the necessity of including update events.

Reply:
What I mean is, if you look at the testcases for UPDATE, e.g. `testUpdateValidationInvalidGroup`, in the testcase you have already created a group and updated a group. So if we craft the testcase more carefully, we can combine  `testUpdateValidationInvalidGroup` and  `testCreateValidationInvalidGroup` in one testcase instead of having two?

Reply:
Thanks, yang. As we synced offline, if we merge create and update, we still need to: 
1. Create an invalid resource
2. Create a valid resource
3. Update the resource to an invalid one

So l will keep it as it is now.